modules/preparing-an-inital-cluster-deployment-for-mce.adoc

```diff
@@ -0,0 +1,401 @@
+// Module included in the following assemblies:
+//
+// * installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-an-inital-cluster-deployment-for-mce_{context}"]
+
+= Preparing an initial cluster deployment for multicluster engine for Kubernetes(MCE) Operator
+
+The following procedure explains how you can:
+
+* Deploy an initial cluster with the Agent-based Installer and use that cluster as the hub cluster.
+* Deploy the MCE Operator.
+* Install the Red Hat Advanced Cluster Management (RHACM) which deploys the assisted service.
+
+.Procedure
+
+. Create a `<manifests_directory>` folder to place a valid `install-config.yaml` and `agent-config.yaml` file. This directory is useed to store all the assets.
```

Comment:
Shouldn't be `<assets folder>`? The name used could sound confusing to me, since there could exists a `cluster-manifests` folder within the `asset-folder` under some circumnstances

Reply:
Noted I will rename where applicabale.

Reply:
I have renamed as <assets_directory> folder as I can see that in docs we have similar refs , is that ok? for example >> https://docs.openshift.com/container-platform/4.11/installing/installing_rhv/installing-rhv-user-infra.html#installation-rhv-creating-install-config-file_installing-rhv-user-infra